src/monster.cpp

```diff
@@ -454,7 +454,7 @@ std::string monster::extended_description() const
     ss << get_tag_from_color( hp_bar.second ) << hp_bar.first << std::endl;
 
     ss << "--" << std::endl;
-    ss << string_format( _( "<dark>%s</dark>" ), type->description.c_str() ) << std::endl;
+    ss << string_format( "<dark>%s</dark>", type->description.c_str() ) << std::endl;
```

Comment:
Since we want interactive language switching, I'd suggest removing `translated_string_reader` from `void mtype::load( JsonObject &jo, const std::string &src )` along with some way of returning the already translated string (like `mtype::get_description()` which would have the macro inside).